### DISCUSSION TITLE
how to define a parameter in jax

### DISCUSSION THREAD

**Discussion by dorost1234** [jax-ml/jax#10390] [Q&A]
Hi
I am not understanding how to define a parameter in jax, i.e., variable for which gradient is computed.
thanks.

**Comment by jakevdp** [comment]
Hi - there is some introductory information on autodiff here: https://jax.readthedocs.io/en/latest/jax-101/01-jax-basics.html#jax-first-transformation-grad and some more advanced treatment at https://jax.readthedocs.io/en/latest/jax-101/04-advanced-autodiff.html and https://jax.readthedocs.io/en/latest/notebooks/autodiff_cookbook.html

If you have any specific questions or points of confusion, please let us know!
